package.json

```diff
@@ -62,6 +62,7 @@
     "eslint-plugin-promise": "^4.0.1",
     "jest": "^26.6.3",
     "jest-diff": "^25.4.0",
+    "jest-extended": "^0.11.5",
```

Comment:
Needed for writing tests in the same format as the new ones written on the typescript branch.